docs/crictl.md

```diff
@@ -13,7 +13,7 @@ or `crictl` please make sure the issue has not already been submitted.
 If you have not already installed crictl please install the version compatible
 with the `cri-containerd` you are using. If you are a user, your deployment
 should have installed crictl for you. If not, get it from your release tarball.
-If you are a developer the current version of crictl is specified [here](./hack/version).
+If you are a developer the current version of crictl is specified [here](./hack/versions).
```

Comment:
It should be `../hack/versions`.

Reply:
@yujuhong Can you also address this?

Reply:
I thought I did, but forgot to amend. Pushed again